### ISSUE TITLE
Données légèrement différentes

### ISSUE THREAD

**Issue by julienvedebe** [bokub/conso-api#33]
Bonjour et tout d'abord un grand merci pour cette API, je viens d'ajouter l'intégration home assistant avec un container en manuel et tout fonctionne correctement.
J'ai bien configuré mon écran énergie et je vois les barres de conso.
En revanche je viens de comparer les valeurs et je trouves des écarts de valeurs entre ce que l'addon m'affiche et l'application Edf&moi.

Exemple hier Samedi 10 fév j'ai 43,78 KWh sur HA et 45,77 KWh sur edf&moi

Mardi 6 Fev 31,62kwh HA et 33,19kwh sur edf&moi

J'ai comparé aussi avec hellowatt les valeurs sont les mêmes que sur l'app Edf.

Est-ce que j'ai loupé quelque chose ? 

J'ai cherché sur les issues passées et je n'ai rien vu de similaire.

Merci de ton aide et encore bravo pour ce super travail !

**Comment by bokub** [comment]
Hello,

Les données viennent de chez Enedis et ne subissent aucune transformation. Peux-tu regarder sur ton espace client Enedis et me dire si les valeurs sont différentes de celles retournées par Conso API ?

**Comment by julienvedebe** [comment]
Je viens de vérifier et j'ai bien une différence entre ce que HA affiche et ce que j'ai dans Enedis suivre mes mesures.

Exemple pour Samedi 10 : 43,78kwh (HA) 45,80 kwh (Enedis)

**Comment by bokub** [comment]
Si tu veux faire l'expérience, tu peux télécharger tes données horaires au format CSV sur le site d'Enedis, et essayer de faire la somme sur une journée pour vérifier si ça correspond à ce qui est affiché sur Home Assistant

**Comment by q1t1** [comment]
Salut @bokub ,
Je n'ai pas encore utilisé tes outils mais j'ai déjà une question ^^

Je remarque que sur le relevé horaire Enedis (avec un pas de 30 minutes), les valeurs sont exactement le double de celle de EDF.

C'est pareil chez toi ?

Merci

**Comment by bokub** [comment]
Je ne suis pas client EDF donc je ne peux pas te répondre, désolé :)

**Comment by q1t1** [comment]
Mais tu ne constates pas des valeurs doublées côté Enedis?

**Comment by bokub** [comment]
Non pas du tout, tu es sûr que le pas est bien de 30 minutes dans les deux sources ?

**Comment by q1t1** [comment]
Oui, et j'ai pu voir que d'autres avaient le même problème que moi...

**Comment by bokub** [comment]
J'ai plutôt tendance à faire confiance aux données d'Enedis, chez moi j'ai un ESP8266 branché au port série de mon Linky, et les données d'Enedis correspondent à ce que je mesure, au centième de kWh

Dans tous les cas il n'y a rien que je puisse faire pour ton problème, Conso API transfère la réponse d'Enedis sans aucune transformation. Si ces donnée sont fausses, l'origine du problème vient forcément de chez eux

**Comment by bokub** [comment]
Attention, mon add-on et conso API ce sont 2 choses différentes

Conso API ne fait aucune transformation, mais l'addon HA Linky fait quelques calculs (pour avoir un pas de 1h par exemple), donc il est possible qu'il y ait des erreurs si j'ai mal codé.

Là tu as créé une issue sur Conso API donc je suppose que c'est de ça dont on parle
